File: data/config-schema/src/main/java/net/corda/schema/configuration/ConfigDefaults.java
package net.corda.schema.configuration;

/**
 * Default configuration values for associated {@link ConfigKeys}
 * <p>
 * NOTE: this is a temporary place for them.
 * Decision to be made on how to specify default values.
 * For example.
 * <ul>
 * <li>Constants</li>
 * <li>HCON config to use as fallback</li>
 * <li>JSON Schema</li>
 * </ul>
 **/
public final class ConfigDefaults {
    private ConfigDefaults() {
    }

    public static final String WORKSPACE_DIR = System.getProperty("java.io.tmpdir") + "/corda/workspace";
    public static final String TEMP_DIR = System.getProperty("java.io.tmpdir") + "/corda/tmp";
}

File: data/config-schema/src/main/kotlin/net/corda/schema/configuration/ConfigDefaults.kt
package net.corda.schema.configuration

/** Default configuration values for associated [ConfigKeys]
 *
 * NOTE: this is a temporary place for them.
 * Decision to be made on how to specify default values.
 * For example.
 *   - Constants
 *   - HCON config to use as fallback
 *   - JSON Schema
 *
 **/
object ConfigDefaults {
    val WORKSPACE_DIR = "${System.getProperty("java.io.tmpdir")}/corda/workspace"
    val TEMP_DIR = "${System.getProperty("java.io.tmpdir")}/corda/tmp"
}
